.github/workflows/ci.yaml

```diff
@@ -1,6 +1,5 @@
-name: Actions
+name: CI
 on:
-  push:
```

Comment:
You could also make this run on push to master and later require it on the release workflow that would eliminate the need to duplicate the jobs in both workflows.
[Here's a question on stackoverflow about it](https://stackoverflow.com/questions/58457140/dependencies-between-workflows-on-github-actions)

Reply:
or better yet make ci a reusable workflow as described [here](https://docs.github.com/en/actions/using-workflows/reusing-workflows)

Reply:
so, trying to make sure i understand what you're saying, this is basically what it was. i'm almost certain what i had before did about the same, but without splitting them was running extra actions that i'm trying to exclude. i think making a better ci reusable workflow is something i want to do that i will probably safe as a larger overhaul and put it over in https://github.com/WFCD/eslint-config so it's more easily shareable